File: app/src/main/java/com/asyabab/endora/data/models/request_payment/AvailableBank.java

package com.asyabab.endora.data.models.request_payment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class AvailableBank implements Serializable {

    @SerializedName("bank_code")
    @Expose
    public String bankCode;
    @SerializedName("collection_type")
    @Expose
    public String collectionType;
    @SerializedName("bank_account_number")
    @Expose
    public String bankAccountNumber;
    @SerializedName("transfer_amount")
    @Expose
    public Integer transferAmount;
    @SerializedName("bank_branch")
    @Expose
    public String bankBranch;
    @SerializedName("account_holder_name")
    @Expose
    public String accountHolderName;
    @SerializedName("identity_amount")
    @Expose
    public Integer identityAmount;

}

File: app/src/main/java/com/asyabab/endora/data/models/payment/checkout/AvailableBank.kt
package com.asyabab.endora.data.models.payment.checkout

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class AvailableBank : Serializable {
    @SerializedName("bank_code")
    @Expose
    var bankCode: String? = null

    @SerializedName("collection_type")
    @Expose
    var collectionType: String? = null

    @SerializedName("bank_account_number")
    @Expose
    var bankAccountNumber: String? = null

    @SerializedName("transfer_amount")
    @Expose
    var transferAmount: Int? = null

    @SerializedName("bank_branch")
    @Expose
    var bankBranch: String? = null

    @SerializedName("account_holder_name")
    @Expose
    var accountHolderName: String? = null

    @SerializedName("identity_amount")
    @Expose
    var identityAmount: Int? = null

}